src/helpers/strings-es.ts

```diff
@@ -84,6 +84,8 @@
     "app-download-app-store-link-alt": "Descargar en la tienda de aplicaciones de Apple",
     "start": "Inicio",
     "resume": "Continuar",
+    "start-survey": "Start Survey",
```

Comment:
I believe if there is no translation in this file, it will default to English. Worth confirming bc this entry does not appear to be translated.

Reply:
I'll substitute in the Spanish translations prior to merging.